src/genie/libs/parser/iosxr/tests/ShowRouteAllSummary/cli/equal/golden_output2_expected.py

```diff
@@ -0,0 +1,182 @@
+expected_output = {
+   "vrf": {
+      "default": {
+         "address_family": {
+            "ipv4_unicast": {
+               "route_source": {
+                  "connected": {
+                     "routes": 11,
+                     "backup": 2,
+                     "deleted": 0,
+                     "memory_bytes": 3120
+                  },
+                  "local": {
+                     "routes": 13,
+                     "backup": 0,
+                     "deleted": 0,
+                     "memory_bytes": 3120
+                  },
+                  "dagr": {
+                     "routes": 0,
+                     "backup": 0,
+                     "deleted": 0,
+                     "memory_bytes": 0
+                  },
+                  "isis": {
+                     "IGP": {
+                        "routes": 195,
+                        "backup": 5,
+                        "deleted": 0,
+                        "memory_bytes": 54160
+                     }
+                  },
+                  "ospf": {
+                     "1": {
+                        "routes": 623,
+                        "backup": 0,
+                        "deleted": 0,
+                        "memory_bytes": 216048
+                     }
+                  },
```

Comment:
I thought we can simply have `ospf 1` and can simplify schema.
```suggestion
                  "ospf 1": {
                     "routes": 623,
                     "backup": 0,
                     "deleted": 0,
                     "memory_bytes": 216048
                  },
```

Reply:
My goal in parsing it that way was that when you have more instances of the same protocol it will be nest an additional dict like this:

```json
"ospf": {
    "13": {
        "routes": 596,
        "backup": 1,
        "deleted": 0,
        "memory_bytes": 144960
    },
    "2": {
        "routes": 11,
        "backup": 0,
        "deleted": 0,
        "memory_bytes": 3088
    }
}
```

So... Are you suggesting to leave the output this way in such cases?

```json
"ospf 13": {
    "routes": 596,
    "backup": 1,
    "deleted": 0,
    "memory_bytes": 144960
},
"ospf 2": {
    "routes": 11,
    "backup": 0,
    "deleted": 0,
    "memory_bytes": 3088
}
```

Taking in consideration that this would apply to all cases in which there is more than one instance of the same protocol.

What is your opinion about?

Ernesto.

Reply:
yes. I knew your goal. I was just suggesting another way to have simpler code/schema. I agreed with your idea. no need to change on this.